### ISSUE TITLE
History list doesn't render updates for labels and favorites

### ISSUE THREAD

**Issue by eezing** [graphql/graphiql#750]
### Current Behavior

1. List not updated on label change.
```gherkin
Given items in history list
When I change label of any item
Then list is not updated
```

2. List not updated on favorites change.
```gherkin
Given items in history list
When I click the star icon on any item
Then list is not updated
```

---

### Possible Solution

https://github.com/graphql/graphiql/blob/9f40b25a576c4e88588aadf7bf13b81fd470457d/src/components/QueryHistory.js#L121

https://github.com/graphql/graphiql/blob/9f40b25a576c4e88588aadf7bf13b81fd470457d/src/components/QueryHistory.js#L136

I think this is the intention:
```javascript
 this.setState({ queries: [...this.historyStore.items, ...this.favoriteStore.items] });
```

**Comment by eezing** [comment]
Already a pull request: #747

**Comment by nikitr** [comment]
@leebyron Is it possible to land the fixes for this issue (fixing favorites and labels)? My team needs this for the work that we are doing with graphiql. Please let me know if we can help in any way to get this code complete.

https://github.com/graphql/graphiql/pull/751

**Comment by wincent** [comment]
@nikitr: sorry but I don't have merge privs on this repo any more.

**Comment by nikitr** [comment]
@wincent Is graphiql being maintained? Do you know who the current maintainers are?

**Comment by wincent** [comment]
@nikitr: I left Facebook in July 2018, so I don't really know anything about who's working on what any more.

**Comment by AGS-** [comment]
Closing this as I have merged fix #747
